Physics/Dynamics/RelativisticBreitWigner.hpp

```diff
@@ -78,14 +79,17 @@ dynamicalFunction(double mSq, double mR, double ma, double mb, double width,
   if (phspFactorSqrtS == std::complex<double>(0, 0))
     return std::complex<double>(0, 0);
 
-  std::complex<double> qRatio =
-      std::pow((phspFactorSqrtS / phspFactormR) * mR / sqrtS, (2 * L + 1));
+  // The each FormFactor includes a term q^L. Therefore only q instead of
+  // q^(2L+1) has to be calculated. Also because phspFactor ~ q/sqrt(s) (see
+  // PDG2018, equation 48.2) the factor \frac{M_R}{\sqrt{s}} can also be omitted
```

Comment:
This is correct to my opinion: `qRatio = qSqrtS / qmR = phspFactorSqrtS / phspFactormR) * sqrtS / mR` 
I'm not sure about the exponent. The implemented form factor is `B_L = q F_L`. Than you still need `qRatio^(2L)`.

Reply:
Equation 48.23 in http://pdg.lbl.gov/2019/reviews/rpp2018-rev-resonances.pdf is what we want to calculate. Because our form factors are different, more precisely `B_L = q^L * F_L` the exponent `2L+1` in equation 48.2, reduces to just `1`.
Now from equation 48.2 we know `rho ~ q / sqrt(s)`. So the term `q_c/q_R * M_R/sqrt(s) = rho_c/rho_R`

Reply:
To my understanding the difference is `B_L = q * F_L`.

Reply:
Can you verify your calculation and post it here if it still does not match

Reply:
I compare `F_L` given in http://pdg.lbl.gov/2019/reviews/rpp2018-rev-resonances.pdf Section 48.2 and the formula implemented in `FormFactor.cpp`.

Reply:
But you see that the pdg `F_L` do not have any q terms in the nominator, while our FormFactor implementation does

Reply:
> To my understanding the difference is `B_L^2 = q * F_L^2`.

Hi, as I understand the F_L in rpp20180rev-resonances and our B_L is related by q^LF_L = B_L
It should be easier find the BW and formfactor formula in this: http://pdg.lbl.gov/2012/reviews/rpp2012-rev-dalitz-analysis-formalism.pdf
Table 2 and Eq. (7) and Eq. (8). I think this kind of BW should be equal to that in rpp-2018-rev-resonances

Reply:
Ah ok I see my mistake. I agree on B_L but still the `qRatio = phspFactorSqrtS / phspFactormR` is not correct. Furthermore, we need to think how the propagate the analytic continuation on phspFactor to `F_L` or `B_L`.

Reply:
Hmm yeah. It seems almost to me, that the K-matrix phase space factor definitions can't be used like this in the BW form. It would be best to use equation 48.24 for the analytic continuation of q. Then the phase space factor rho should not be used in the BW and Flatte anymore.

Reply:
Or can we keep rho as it is except that it does not use the q anymore, but uses this rho hat definition. Then we simply define q ~ rho * sqrt(s)? But here we would have to use a complex value for s < 0 [so sqrt(s) = i * sqrt(|s|) for s < 0]